xrdsst/controllers/endpoint.py

```diff
@@ -56,20 +72,82 @@ def remote_add_service_endpoints(self, ss_api_config, security_server_conf, clie
+                                                                    + endpoint_conf["method"] + " " + endpoint_conf["path"] + "' for service '" + service_description.id + "', no valid candidate found")
+                                        else:
+
+                                            endpoint = [e for e in service_description.services[0].endpoints if e.method == endpoint_conf["method"] and e.path == endpoint_conf["path"]]
+                                            if len(endpoint) == 0:
+                                                BaseController.log_info(
+                                                    "Error adding client access rights '" + access + "' for the endpoint '" + endpoint_conf["method"] + " " + endpoint_conf["path"] + "' for service '" + service_description.id + "', endpoint not found")
+                                            else:
+                                                try:
+                                                    endpoints_api = EndpointsApi(ApiClient(ss_api_config))
+                                                    response = endpoints_api.add_endpoint_service_clients(endpoint[0].id, body=ServiceClients(items=candidate))
+                                                    if response:
+                                                        BaseController.log_info("Added client access rights: '"+candidate[0].id + "'for endpoint '" + endpoint[0].method +
+                                                                                "' '" + endpoint[0].path + "' in service '" + service_description.services[0].id + "'")
+                                                except ApiException as err:
+                                                    if err.status == 409:
+                                                        BaseController.log_info(
+                                                            "Added client access rights: '" + candidate[
+                                                                0].id + "'for endpoint '" + endpoint[0].method +
+                                                            "' '" + endpoint[0].path + "' in service '" +
+                                                            service_description.services[0].id + "' already added")
+                                                    else:
+                                                        BaseController.log_api_error('EndpointsApi->add_endpoint_service_clients', err)
+
+                            except ApiException as find_err:
+                                BaseController.log_api_error('ClientsApi->get_client_service_description', find_err)
+                except ApiException as find_err:
+                    BaseController.log_api_error('ClientsApi->get_client_service_description', find_err)
+        except ApiException as find_err:
+            BaseController.log_api_error('ClientsApi->find_clients', find_err)
+
+
+    def getServicesDescription(self, config):
```

Comment:
method "getServicesDescription()" would be ok for Java, but Python uses snake-case, so probably get_services_description()" would be better.(https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)